### ISSUE TITLE
Enhancement: Deployment Docs and 1-Click Deployment Button

### ISSUE THREAD

**Issue by platform-kit** [OneUptime/oneuptime#1165]
**Is your feature request related to a problem? Please describe.**
Docs don't mention anything about deploying to production. 

**Describe the solution you'd like**
I would like to see a fully-realized, thorough, step-by-step doc that does not assume anything about the end user's level of knowledge or skip over anything, from first install (git clone) to deployment.  Additionally, to mitigate the technical hurdles tha deployment might represent for the less technically inclined, it would be great to have a "Deply to Render" button.

See: https://docs.render.com/deploy-to-render#:~:text=Once%20you%20add%20the%20Deploy,and%20without%20any%20additional%20work.

Basically all that is required is a `render.yaml` file that describes the docker setup command, the database, the port that the web server runs on, and a handful of other things.
